docs/tutorials/tutorial-0.rst

```diff
@@ -45,6 +45,8 @@ Check out what the provided ``tutorial_0/app.py`` file contains:
 
 This won't do much as it is, but we can make it useful. 
 
+IF LINUX import __main__ script
```

Comment:
My intent was to tell the user to import the main script into the app.py
or for example in my code at the top of the file I have:

import __main__